omero-dev-runtests/Dockerfile

```diff
@@ -0,0 +1,114 @@
+FROM openmicroscopy/omero-ssh-systemd
+MAINTAINER ome-devel@lists.openmicroscopy.org.uk
+
+# To avoid error: sudo: sorry, you must have a tty to run sudo
+RUN sed -i -re 's/Defaults\s+requiretty.*//' /etc/sudoers
+
+# OMERO.py prerequesties
+RUN yum install -y epel-release && \
+    yum clean all
+
+RUN yum install -y unzip && \
+    yum clean all
+
+RUN yum install -y \
+    python-pip python-devel python-virtualenv \
+    numpy scipy python-matplotlib python-tables && \
+    yum clean all
+  
+RUN curl -o /etc/yum.repos.d/zeroc-ice-el7.repo \
+    http://download.zeroc.com/Ice/3.5/el7/zeroc-ice-el7.repo
+
+RUN yum install -y java-1.8.0-openjdk-devel \
+    yum clean all
+
+RUN yum install -y ice ice-python-devel ice-java-devel ice-servers && \
+    yum clean all
+
+RUN yum install -y git \
+    zlib-devel \
+    libjpeg-devel \
+    gcc && \
+    yum clean all
+
+RUN yum install -y python-pillow \
+    yum clean all
+
+RUN yum -y install \
+    http://yum.postgresql.org/9.4/redhat/rhel-7-x86_64/pgdg-centos94-9.4-2.noarch.rpm \
```

Comment:
Also missing `&&`. If you want you could merge the next line in too e.g.

```
RUN yum -y install \
    http://yum.postgresql.org/9.4/redhat/rhel-7-x86_64/pgdg-centos94-9.4-2.noarch.rpm && \
    yum -y install postgresql94-server postgresql94 && \
    yum clean all
```

and similarly for some of the other blocks, but that's up to you.